keras/layers/recurrent.py

```diff
@@ -365,6 +376,8 @@ def step(self, x, states):
             h = K.dot(x * B_W, self.W) + self.b
 
         output = self.activation(h + K.dot(prev_output * B_U, self.U))
+        if 0 < self.zoneout_h < 1:
```

Comment:
Interesting border case, it could be useful for comparison purposes, but on the other hand it gets pretty similar to a time-distributed dense layer.